### DISCUSSION TITLE
Crafty Controller

### DISCUSSION THREAD

**Discussion by Atlantis-23** [selfhst/icons#152] [New Icon Request]
Crafty Controller is a self-hosted Minecraft server dashboard. Attached is the highest quality icon that is from their gitlab assets.

![crafty-logo-square-1024](https://github.com/user-attachments/assets/a1cc17a3-6f08-40fd-9b14-fc8a5a53319f)

The official asset icons can be found here on their gitlab: https://gitlab.com/crafty-controller/crafty-4/-/tree/master/app/frontend/static/assets/images

**Comment by shollyethan** [comment]
Crafty Controller has been added - thanks!
